pkg/controller/vc_config_controller.go

```diff
@@ -82,6 +82,12 @@ func (v *vcConfigController) processVcConfigSecretItem(key string) error {
 		return err
 	}
 	v.logger.Debug("Successfully retrieved latest vSphere VC credentials.")
+	err = utils.RetrieveVddkLogLevel(ivdParams, v.logger)
+	if err != nil {
+		v.logger.WithError(err).Error("Failed to retrieve vddk log level")
+		return err
+	}
+	v.logger.Debug("Successfully retrieved vddk log level")
```

Comment:
We don't plan to support reloading VDDK log level in the run time. As far as we observe, it doesn't work somehow. Do we really need this piece of code?

Reply:
Remove this piece of code. Only the first call for vddk initial works as expected, which is called when starting server.